modules/generic_worker/templates/generic-worker.config.multiuser.erb

```diff
@@ -0,0 +1,42 @@
+{
+  "accessToken": "<%= @taskcluster_access_token %>",
+  "cachesDir": "<%= @caches_dir %>",
+  "certificate": "",
+  "checkForNewDeploymentEverySecs": 0,
+  "cleanUpTaskDirs": true,
+  "clientId": "<%= @taskcluster_client_id %>",
+  "deploymentId": "",
+  "downloadsDir": "<%= @downloads_dir %>",
+  "idleTimeoutSecs": 345600,
+  "instanceId": "",
+  "instanceType": "",
+  "livelogExecutable": "livelog",
```

Comment:
Is livelog deprecated? Maybe in a follow-up change we can remove this and the livelog install since we have wst* set and working.

Reply:
The livelog executable is still needed when using websocktunnel, but the livelog secrets are not needed, as access to the livelog stream (provided by the livelog executable) is provided via the tunnel, rather than a direct client-to-worker ssl connection that was previously used. The former secrets enabled the service to be hosted over https on the worker directly, and the livelog secret was used in tandem with the stateless dns server in order to provide hashed hostnames for the workers.

Reply:
thx

Reply:
for the explanation 👍  I won't expect to remove the livelog binary now